examples/deletecard/app.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import Announcer from '../../src/Announcer';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      announcement: '',
+      cards: [
+        {name:'Kitten McMittens'},
+        {name:'Cindy Clawford'},
+        {name:'Guy Purri'},
+        {name:'Chester Fishington'},
+        {name:'Bing Clawsby'},
+      ]
+    }
+
+    this.deleteElement = this.deleteElement.bind(this);
+  }
+
+  deleteElement(id, card) {
+    this.setState({
+      cards: this.state.cards.filter(function (e, i) {
+        return i !== id;
+      })
+    });
+
+    this.pushAnnouncement(`${card.name} has been removed.`);
+  }
+
+  pushAnnouncement(announcement) {
+    this.setState(prevState => ({
+      announcement: announcement
+    }));
+  }
+
+  render() {
+
+    var cards = this.state.cards.map(function(card, i) {
+      return <div className="card" key={ i }>
+        <img src="http://lorempixel.com/output/cats-q-c-200-200-6.jpg"/>
```

Comment:
I'm not sure why but these images aren't loading until I manually click and resolve the URL